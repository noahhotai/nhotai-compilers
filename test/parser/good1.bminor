x:integer;
y:integer;
z:integer;