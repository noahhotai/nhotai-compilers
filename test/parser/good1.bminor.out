IDENTIFIER     x
COLON :
INT            integer
SEMICOLON      ;
IDENTIFIER     y
COLON :
INT            integer
SEMICOLON      ;
IDENTIFIER     z
COLON :
INT            integer
SEMICOLON      ;
Program parsed successfully.
